### PR TITLE
Add speed search to the Preview view.

### PR DESCRIPTION
**Pull request by scheglov**

### CODE REVIEW

File: src/io/flutter/preview/PreviewView.java [hunk 1]
```diff
@@ -264,6 +279,42 @@ Icon getIcon() {
     return icon;
   }
 
+  /**
+   * Return the string that is suitable for speed search. It has every name part separted so that we search only inside individual name
+   * parts, but not in their accidential concatenation.
+   */
+  @NotNull
+  String getSpeedSearchString() {
```
Comment (devoncarew) [inline src/io/flutter/preview/PreviewView.java]: private?

Comment (scheglov) [reply src/io/flutter/preview/PreviewView.java]: In Java private is stronger than in Dart, it prevents access even for classes in the same file. So, default visibility.

**Comment by scheglov** [comment]
I opened a new issue https://github.com/flutter/flutter-intellij/issues/1697 for speed search in Flutter Inspector.

**Comment by pq** [comment]
Late to the party (sorry) but 🎉 🎉 🎉 !

**Pull request merged**